lua/astrocommunity/editing-support/nvim-ts-rainbow2/nvim-ts-rainbow2.lua

```diff
@@ -0,0 +1,15 @@
+return {
+  "nvim-treesitter/nvim-treesitter",
+  dependencies = { "HiPhish/nvim-ts-rainbow2" },
+  opts = { rainbow = { enable = true } },
+  config = function(_, opts)
+    -- HACK: https://github.com/p00f/nvim-ts-rainbow/issues/112#issuecomment-1310835936
+    vim.api.nvim_create_autocmd({ "BufWritePost", "FocusGained" }, {
+      callback = function()
+        vim.cmd "TSDisable rainbow"
+        vim.cmd "TSEnable rainbow"
+      end,
+    })
+    require("nvim-treesitter.configs").setup(opts)
+  end,
+}
```

Comment:
```suggestion
return {
  "nvim-treesitter/nvim-treesitter",
  dependencies = {
    "HiPhish/nvim-ts-rainbow2",
    config = function()
      -- HACK: https://github.com/p00f/nvim-ts-rainbow/issues/112#issuecomment-1310835936
      vim.api.nvim_create_autocmd({ "BufWritePost", "FocusGained" }, {
        callback = function()
          vim.cmd "TSDisable rainbow"
          vim.cmd "TSEnable rainbow"
        end,
      })
    end,
  },
  opts = { rainbow = { enable = true } },
}
```

I think this is a better approach to adding that `autocmd`

Reply:
Done